File: app-marketplace/marketplace-backend/Backend.Domain/DTO/UserCreateDTO.cs
using Backend.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Backend.Domain.DTO
{
    public class UserCreateDTO
    {
        public required string Password { get; set; }
        public required string Name { get; set; }
        public required string Email { get; set; }
        public required string PhoneNumber { get; set; }
        public required string DocumentNumber { get; set; }
        public long ZipCode { get; set; }
        public required string AddressName { get; set; }
        public long Number { get; set; }
        public required string Neighborhood { get; set; }
        public required string City { get; set; }
        public required string State { get; set; }
        public required string BirthdayDate { get; set; }
        public required string ClientName { get; set; }
        
        public int ProfileId { get; set; }
        public bool IsNewClient { get; set; }
    }
}

File: app-marketplace/marketplace-frontend-fix/src/app/models/DTO/UserCreateDTO.ts
export interface UserCreateDTO {
    password: string,
    name: string,
    email: string,
    phoneNumber: string,
    documentNumber: string,
    zipCode: string,
    addressName: string,
    number: string,
    neighborhood: string,
    city: string,
    state: string,
    profileId: 0,
    birthdayDate:string,
    isNewClient : true,
    clientName:string,
    imageGuid : string,
    companyName : string
}